cpp/test/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+# keep the files in alphabetical order!
+add_executable(test_rapids_triton
```

Comment:
Note: I'm about to do a PR to cuML to have separate executables for each test file, not sure if we want to do the same here? If so, that could be a follow up issue/PR

Reply:
Hmmm... That's a good question. I'm on the fence on that one. On the one hand, I generally like the modularity of separate test files, but on the other, the tests are *way* faster to run than for cuML, so I'm not sure it buys us a ton. Let me make a follow up issue to at least consider it.